### PR TITLE
classes and functions for grant fields

### PR DESCRIPTION
**Pull request by colbypexton**
Story=

As a manager, I want to update information about relevant grants, including due dates and requirements, so that our grant applications are accurate and up-to-date.

Acceptance Criteria=
1. create a field class with constructor & getter methods
2. create a function that passes SQL row data to the constructor and returns a new field

Log=
-create a Field class
-created getter methods for the Field class
-created a function that passes SQL data into a new Field object

### CODE REVIEW

File: database/dbEvents.php [hunk 1]
```diff
@@ -161,6 +162,15 @@ function fetch_events_on_date($date) {
     return $events;
 }
 
+function make_field($result_row){
```
Comment (ColinRyan03) [inline database/dbEvents.php:171]: Consider adding a comment explaining the function and its purpose

File: database/dbEvents.php [hunk 2]
```diff
@@ -161,6 +162,15 @@ function fetch_events_on_date($date) {
...
+    $field = new Field(
+        null,
+        $result_row['name'],
```
Comment (ColinRyan03) [inline database/dbEvents.php:174]: Spacing might not be necessary here (could have arguments just on one line)

Comment (colbypexton) [reply database/dbEvents.php:174]: Good catch. This is formatted based on another function similar to one for a different object. Let's discuss general formatting next meeting

File: database/dbEvents.php (hunk 1 above)
Comment (colbypexton) [reply database/dbEvents.php:171]: Will do!

**Pull request merged**